CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Bugfix: Don't create native messaging manifest file if browser directory doesn't exist. (#6116)
 - Dev: Conan will no longer generate a `CMakeUserPresets.json` file. (#6117)
+- Minor: Make paused chat indicator more visible. (#6123)
```

Comment:
Move to the top of the changelog entries. They're generally sorted like this:
Major
Minor
Bugfix
Dev